source/release-notes.txt

```diff
@@ -10,6 +10,30 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+Relational Migrator 1.1.3
+-------------------------
+
+*Released July 26, 2023*
+
+New features:
+
+- Added support for connecting to a SQL Server Named Instance.
+- Added collection names to JSON schema files created from the :guilabel:`Code Generation` screen.
+
+Bug Fixes:
+
+- Fixed an issue where data verification after a sync job would sometimes 
+  abort or show as not started.
```

Comment:
(inclusive language / global audience / [jargon](https://www.mongodb.com/docs/meta/style-guide/terminology/terms-for-global-audience/#use-jargon-with-care))
Let's use stops, ends, or quits instead of aborts.

Reply:
Updated.